recipes/himalaya/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.1)
+project(PackageTest CXX)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+add_executable(test_package test_package.cpp)
+target_link_libraries(test_package CONAN_PKG::himalaya)
```

```suggestion
target_link_libraries(test_package CONAN_PKG::himalaya)
set_property(TARGET test_package PROPERTY CXX_STANDARD 14)
```

https://github.com/Himalaya-Library/Himalaya/blob/4.1.0/CMakeLists.txt#L24